support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep.java

```diff
@@ -41,11 +39,15 @@
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
 
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
 public class StashStep extends AbstractStepImpl {
 
     private final @Nonnull String name;
     private @CheckForNull String includes;
     private @CheckForNull String excludes;
+    private boolean useDefaultExcludes = true;
```

Comment:
:bug: need to also add to `config.jelly`

```xml
<f:entry field="useDefaultExcludes" title="…something…">
    <f:checkbox default="true"/>
</f:entry>
```

and a matching `help-useDefaultExcludes.html`.